feat: improve plugin first screen

girishpanchal30 opened the pull request:
### All Submissions:

* [x] Have you followed the [Optimole Contributing guideline](https://github.com/Codeinwp/optimole-wp/blob/master/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change? 
 
### Changes proposed in this Pull Request:
Improve the account connections page layout as suggested [here](https://github.com/Codeinwp/optimole-service/issues/1335#issuecomment-2687267604).

![image](https://github.com/user-attachments/assets/8ca00a80-0f70-4e2d-aea0-3ea90e9ebb66)

Closes https://github.com/Codeinwp/optimole-service/issues/1335

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

abaicus reviewed (changes requested):
@girishpanchal30 let's refrain from using classes with dynamically defined values like `bg-[#f0f6fc]`, or `text-[#646970]`, `text-[#2271b1]`, etc. 

We can surely either match the colors to some variation of the custom-defined colors, or predefined tailwind values

girishpanchal30 commented:
@abaicus Thank you for the review! I've fixed the feedback. Please review and let me know if any further changes are needed.

abaicus commented on assets/src/dashboard/parts/connect/index.js:
```diff
@@ -187,6 +199,14 @@ const ConnectLayout = () => {
 						/>
 					</div>
+
+					<div className="flex py-3">
```
We can also add a `items-center` class on all three items in this list so that the checkmark is centered. Maybe also map them to an array with the contents we can loop through and output them so we don't have duplicated markup.

girishpanchal30 replied:
@abaicus fixed all suggestions

abaicus reviewed (changes requested):
@girishpanchal30 I meant using variables that already exist in the config, instead of introducing new ones either inline, or in the config itself. We can ballpark the design, and don't need it to be pixel-perfect to the artifact

- `slate-gray` can be matched to `gray-500`;
- The value of `text-s` is actually the font-size of the in the admin, so we could simply not add it;
- `text-15`, we can simply use `text-sm` which is 1px lower;

abaicus commented:
@girishpanchal30 I see there are still some leftover variables lef tin tailwind config. Can't we replace text-26 with text-2xl or something to that affect?

`text-s` is also leftover.

girishpanchal30 commented:
> @girishpanchal30 I see there are still some leftover variables lef tin tailwind config. Can't we replace text-26 with text-2xl or something to that affect?

Removed text-26 class

> text-s is also leftover

I’ve already removed it from where it was previously used. Could you please recheck with the latest commit?

abaicus commented:
Thanks @girishpanchal30 ! Looks good! 👍🏻
